src/api/util/helpers.ts

```diff
@@ -138,6 +145,42 @@ export function parseBrc20Balances(items: DbBrc20Balance[]): Brc20BalanceRespons
   }));
 }
 
+export function parseBrc20Activities(items: DbBrc20Activity[]): Brc20ActivityResponse[] {
+  return items.map(i => {
+    const a: Partial<Brc20ActivityResponse> = {
```

Comment:
Instead of using a Partial and checking types twice below you could use a generic object and finally incorporate it into the final correct type:
```typescript
{
  ...obj,
  max_supply: i.deploy_max,
  // etc
}
```